app/validation/validators.py

```diff
@@ -68,25 +68,31 @@ class NumberRange(object):
         The maximum value of the number. If not provided, maximum value
         will not be checked.
     """
-    def __init__(self, minimum=None, maximum=None, messages=None):
+    def __init__(self, minimum=None, maximum=None, messages=None, format_type=None):
         self.minimum = minimum
         self.maximum = maximum
         if not messages:
             messages = error_messages
         self.messages = messages
+        self.format_type = format_type
 
     def __call__(self, form, field):
         data = field.data
         error_message = None
         if data is not None:
             if self.minimum is not None and data < self.minimum:
-                error_message = self.messages['NUMBER_TOO_SMALL'] % dict(min=format_number(self.minimum))
+                error_message = self.messages['NUMBER_TOO_SMALL'] % dict(min=self.format_min_max(self.minimum))
             elif self.maximum is not None and data > self.maximum:
-                error_message = self.messages['NUMBER_TOO_LARGE'] % dict(max=format_number(self.maximum))
+                error_message = self.messages['NUMBER_TOO_LARGE'] % dict(max=self.format_min_max(self.maximum))
 
             if error_message:
                 raise validators.ValidationError(error_message)
 
+    def format_min_max(self, value):
+        if self.format_type == 'Currency':
+            return format_currency(value)
```

Comment:
Should we not also pass in the "currency" field to ensure the error message matches the answer asked? At the moment this will always default to GBP.